checks/apps/pytorch/mlperf_storage_ce.py

```diff
@@ -99,10 +95,14 @@ def setup_test(self):
         self.num_tasks = self.mlperf_data.num_tasks
         self.env_vars = self.mlperf_data.env_vars
         self.workload = self.mlperf_data.workload
-        self.container_mounts = self.mlperf_data.container_mounts
         self.container_workdir = self.mlperf_data.container_workdir
         num_files = self.mlperf_data.num_files
         accelerator_type = self.mlperf_data.accelerator_type
+        self.container_mounts = [
+            f'{self.stagedir}/unet3d.yaml:'
+            '/workspace/storage/storage-conf/workload/unet3d_h100.yaml',
+            f'{self.mlperf_data.storage}:/mlperf_storage',
+        ]
```

Comment:
```suggestion
        self.container_mounts = self.mlperf_data.container_mounts
```
no reason to repeat yourself ?

Reply:
Theofilos Manitaras
It's that the stage dir is going to be the one from the fixture, not the one of the test